Formula/monitoring-plugins.rb

```diff
@@ -0,0 +1,39 @@
+class MonitoringPlugins < Formula
+  desc "Plugins for nagios compatible monitoring systems"
+  homepage "https://www.monitoring-plugins.org"
+  url "https://www.monitoring-plugins.org/download/monitoring-plugins-2.2.tar.gz"
+  sha256 "296a538f00a9cbef7f528ff2d43af357a44b384dc98a32389a675b62a6dd3665"
+
+  depends_on "openssl"
+  depends_on "postgresql" => :optional
+  depends_on :mysql => :optional
+
+  conflicts_with "nagios-plugins", :because => "nagios-plugins ships their plugins to the same folder."
```

Comment:
It’d be great if you could make a commit that adds a similar line in the `nagios-plugins.rb` formula.